crates/serde/src/lib.rs

```diff
@@ -18,6 +18,8 @@ use serde::Serializer;
 mod bool;
 pub use self::bool::*;
 
+pub mod optional;
```

Comment:
I would prefer private module and re-export the function

Reply:
Addressed in https://github.com/alloy-rs/alloy/pull/955/commits/fdd21940e2046775d223b23ffa02cc67f855379d